StitchCore/Sources/StitchCore/StitchClientConfiguration.swift

```diff
@@ -1,17 +1,54 @@
+/**
+ * :nodoc:
+ * Properties representing the configuration of a client that can communicate with MongoDB Stitch.
+ */
 public protocol StitchClientConfiguration {
+    /**
+     * The base URL of the Stitch server that the client will communicate with.
+     */
     var baseURL: String { get }
+
+    /**
+     * The underlying storage for authentication info.
```

Comment:
[nit] It's kind of more than just for auth info. We store device details too. Maybe `The underlying storage for persisting authentication and app state.`